src/EmailRecoveryFactoryEasy.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import { Create2 } from "@openzeppelin/contracts/utils/Create2.sol";
+import { EmailRecoveryManager } from "./EmailRecoveryManager.sol";
+import { UniversalEmailRecoveryModule } from "./modules/UniversalEmailRecoveryModule.sol";
+import { EmailRecoveryModule } from "./modules/EmailRecoveryModule.sol";
+
+contract EmailRecoveryFactoryEasy {
+    address public immutable VERIFIER;
+    address public immutable DKIM_REGISTRY;
+    address public immutable EMAIL_AUTH_IMPL;
+
+    event ZkEmailRecoveryModuleCreated(
+        address emailRecoveryModule, address emailRecoveryManager, address subjectHandler
+    );
+
+    event ZkEmailRecoveryManager(
+        address emailRecoveryManager, address subjectHandler, address forModule
+    );
+
+    constructor(address verifier, address dkimRegistry, address emailAuthImpl) {
+        VERIFIER = verifier;
+        DKIM_REGISTRY = dkimRegistry;
+        EMAIL_AUTH_IMPL = emailAuthImpl;
+    }
+
+    function deployForModule(
+        address module,
+        bytes calldata subjectHandlerBytecode,
+        bytes32 subjectHandlerSalt,
+        bytes32 recoveryManagerSalt
+    )
+        external
+        returns (address emailRecoveryManager, address subjectHandler)
+    {
+        // Deploy subject handler
+        subjectHandler = Create2.deploy(0, subjectHandlerSalt, subjectHandlerBytecode);
+        // Deploy recovery manager
+        emailRecoveryManager = address(
+            new EmailRecoveryManager{ salt: recoveryManagerSalt }(
+                VERIFIER, DKIM_REGISTRY, EMAIL_AUTH_IMPL, subjectHandler
+            )
+        );
+
+        emit ZkEmailRecoveryManager(emailRecoveryManager, subjectHandler, module);
+
+        // Initialize recovery manager with module address
+        EmailRecoveryManager(emailRecoveryManager).initialize(module);
+    }
+
+    function predictAddressesForModule(
+        bytes32 subjectHandlerSalt,
+        bytes calldata subjectHandlerBytecode,
+        bytes32 recoveryManagerSalt
+    )
+        external
+        view
+        returns (address subjectHandler, address emailRecoveryManager)
```

Comment:
`emailRecoveryManager` address would be used in the constructor of the 3rd party recovery module